pkg/webhook/stats_reporter.go

```diff
@@ -16,31 +16,25 @@ const (
 )
 
 var (
-	req metric.Float64ValueRecorder
+	req instrument.Float64Histogram
 	// if service.name is not specified, the default is "unknown_service:<exe name>"
 	// xref: https://opentelemetry.io/docs/reference/specification/resource/semantic_conventions/#service
 	labels = []attribute.KeyValue{attribute.String("service.name", "webhook")}
 )
 
-// reporter implements StatsReporter.
-type reporter struct {
-	metric.Meter
-}
+func registerMetrics() error {
```

Comment:
Does this have to be global?

Reply:
It's only used in webhook and register is called when pod mutator is created, so we don't need this to be global.